src/components/shapes/label_texttemplate.js

```diff
@@ -0,0 +1,28 @@
+'use strict';
+
+module.exports = {
+    shapeLabelTexttemplateVars: {
```

Comment:
Let's drop `shapeLabelTexttemplateVars` and simplify this export now that there is only one object being exported.
```js
module.exports = {
  x0: ...,
  x1: ...,
  ...
  
  Then the `require` statements would be
  ```js
  var shapeLabelTexttemplateVars = require('./label_texttemplate');
  ``` 
};